packages/perseus/src/components/sortable.tsx

```diff
@@ -66,8 +68,35 @@ class Placeholder extends React.Component<any> {
     }
 }
 
-type DraggableProps = any;
-type DraggableState = any;
+type DraggableProps = {
+    content: string;
+    endPosition: {left: number; top: number} | Record<string, never>;
+    includePadding: boolean;
+    layout: Layout;
+    width?: number;
+    height?: number;
+    margin?: string;
+    onAnimationEnd: () => void;
+    onMouseDown: () => void;
+    onMouseMove: () => void;
+    onMouseUp: () => void;
+    onRender: () => void;
+    type: typeof STATIC | typeof DRAGGING | typeof ANIMATING | typeof DISABLED;
```

Comment:
I checked that `enum`s work with `flowgen` and they do.  I think this would make a great enum.